README.md

```diff
@@ -1,5 +1,10 @@
 # Kubernetes vSphere Cloud Provider
 
+![GitHub release (latest SemVer including pre-releases](https://img.shields.io/github/v/release/kubernetes/cloud-provider-vsphere?include_prereleases)
```

Comment:
Curious about this link to img.shields.io, what's it for?

Reply:
it displays our latest release version of cpv